packs/pathfinder-society-boons/exotic-edge.json

```diff
@@ -13,6 +13,10 @@
         "description": {
             "value": "<p><strong>Cost</strong>: 4 AcP</p><hr /><p>By traveling so widely, you have developed an intuitive talent for using esoteric techniques. Once per adventure, you can activate this boon as a free action to apply these techniques, gaining a +1 circumstance bonus to either all of your attack rolls or all of your skill checks for 1 round.</p>"
         },
+        "frequency": {
+            "max": 1,
+            "per": "day"
+        },
```

Comment:
Not sure about setting this as per day since I think some PFS scenarios take multiple days.